Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLInterfaceExporter.swift

```diff
@@ -0,0 +1,150 @@
+                                            <script
+                                              crossorigin
+                                              src="https://unpkg.com/react-dom/umd/react-dom.production.min.js"
+                                            ></script>
+                                            <script
+                                              crossorigin
+                                              src="https://unpkg.com/graphiql/graphiql.min.js"
+                                            ></script>
+                                            <script>
+                                              const graphQLFetcher = graphQLParams =>
+                                                fetch('/graphql', {
+                                                  method: 'post',
+                                                  headers: { 'Content-Type': 'application/json' },
+                                                  body: JSON.stringify(graphQLParams),
+                                                })
+                                                  .then(response => response.json())
+                                                  .catch(() => response.text());
+                                              ReactDOM.render(
+                                                React.createElement(GraphiQL, { fetcher: graphQLFetcher }),
+                                                document.getElementById('graphiql'),
+                                              );
+                                            </script>
+                                          </body>
+                                        </html>
+                                        """
+
+        return Vapor.Response(status: .ok, headers: ["Content-Type": "text/html"], body: html)
+    }
+
+    private func graphqlServer(_ req: Vapor.Request) throws -> EventLoopFuture<String> {
+        guard let body = req.body.string else {
+            throw NetworkError.noBody
+        }
+
+        // Create Swift dict
+        var query = "{}"
+        let data = body.data(using: .utf8)!
+        do {
+            if let jsonArray = try JSONSerialization.jsonObject(with: data, options: .allowFragments) as? [String: AnyObject] {
+                if let q = jsonArray["query"] as? String {
+                    query = q
+                } else {
+                    print("jsonArray doesn't have query parameter")
+                }
+
+            } else {
+                print("bad json")
+                throw NetworkError.noBody
+            }
+        } catch let error as NSError {
+            print(error)
+            throw NetworkError.noBody
+        }
+
+        return try graphql(schema: self.schema,
+                request: query,
+                context: req,
+                eventLoopGroup: req.eventLoop).map { result -> String in
+            result.description
+        }
```

Comment:
Please try to match the formatting we use in Apodini:
```suggestion
        try graphql(
            schema: self.schema,
            request: query,
            context: req,
            eventLoopGroup: req.eventLoop
        )
            .map { result -> String in
                result.description
            }
```
or alternatively
```suggestion
        try graphql(schema: self.schema,
                    request: query,
                    context: req,
                    eventLoopGroup: req.eventLoop)
            .map { result -> String in
                result.description
            }
```